integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -235,13 +242,17 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
 
   @Override
   public TableOperations newTableOps(TableIdentifier tableIdentifier) {
-    return OpenHouseTableOperations.builder()
-        .tableIdentifier(tableIdentifier)
-        .fileIO(fileIO)
-        .tableApi(tableApi)
-        .snapshotApi(snapshotApi)
-        .cluster(cluster)
-        .build();
+    FileIO tableOperationsLocalFileIO = loadFileIO(properties);
```

Comment:
can we also change the signature of loadFileIO to following:
```
protected FileIO loadFileIO(TableIdentifier tableIdentifier, Map<String, String> properties) {
```

reason:
Each table can have its own storage, table1 -> hdfs, table2 -> s3, table3 -> adls . We'd need to instantiate/load the right FileIO per table and properties.